### PR TITLE
Added Return Type Declarations to functions.php

### PR DESCRIPTION
**Pull request by jlolzzz**
Added return type declaration to functions in functions.php file. Ran on xampp server with API key and didn't receive any errors. Only concern was function shortNumber() as I was concerned it might return an error for two return types which is not supported (unless using union types in php8 - something I just learned). added "string" type to that function, which is was it appears to be read/used as. Let me know if there are any issues. Thanks!

### CODE REVIEW

File: web/functions.php [hunk 1]
```diff
@@ -26,7 +26,7 @@ function validateParam($name, $pattern, $defaults)
 }
 
 // Use YouTube Data API and Shields.io to get an image URL
-function getShieldURL($query, $defaults)
+function getShieldURL($query, $defaults): array
```
Comment (DenverCoder1) [inline web/functions.php]: `getShieldURL()` returns a string type as well

**Review by DenverCoder1 — CHANGES_REQUESTED** [review]
👍 Looks good. Only 1 requested change.

**Comment by jlolzzz** [comment]
Ok made the correction. I am not sure if there's something else that I need to do after. I pushed the change, I haven't really used GitHub this much before.

**Review by DenverCoder1 — APPROVED** [review]
👍 Awesome! Looks good, I will merge these changes.

**Pull request merged**